src/pages/Account/Account.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import {
+  Nav,
+  NavItem,
+  NavLink,
+  Container,
+  Col,
+  Row,
+} from 'reactstrap';
+import { Route, Redirect, Switch } from 'react-router';
+import { NavLink as RRNavLink } from 'react-router-dom';
+import NotFound from 'components/routes/NotFound';
+import AccountGeneral from './AccountGeneral';
+import AccountPayment from './AccountPayment';
+import AccountSecurity from './AccountSecurity';
+import AccountVerification from './AccountVerification';
+import { GET_ACCOUNT_PAGE } from 'networking/users';
+import { Query } from 'react-apollo';
+import AudioLoading from 'shared/loading/AudioLoading';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+
+const Account = () => {
+  return (
+    <Query query={GET_ACCOUNT_PAGE}>
+      {({ loading, error, data }) => {
+        if (loading) {
+          return <AudioLoading height={48} width={96} />;
+        }
+        if (error || !data) {
+          return <h1>{error ? error.message : 'Error / No Data'}</h1>;
+        }
+        
+        const { creditBalance, user } = data;
+        return (
+          <Container>
+            <h1>Profile</h1>
+            <hr />
+            <Nav tabs>
+              <NavItem>
+                <NavLink
+                  activeClassName="active"
```

Comment:
should we move these details to an array of objects?

Reply:
done